pkgs/applications/networking/feedreaders/feedreader/default.nix

```diff
@@ -7,7 +7,7 @@ let
   pname = "FeedReader";
   version = "2.2";
 in stdenv.mkDerivation {
-  name = "${pname}-${version}";
+  inherit pname version;
```

Comment:
It case like this one, both ```let```/```in``` and ```inherit``` are not needed (although sometimes it would require ```rec```)

Reply:
Yeah, I just `sed`ded for this round, but I'll do some more manual review.

Reply:
... Actually, I'll guess I'll extend [nix-linter](https://github.com/synthetica9/nix-linter) to do that review for me!